.github/workflows/docs-check.yml

```diff
@@ -14,18 +14,13 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
         with:
           python-version: 3.9
 
       - name: Install dependencies
         run: pip install sphinx sphinx_rtd_theme setuptools-rust
 
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
```

Comment:
is there a reason why this one is deleted as well?

Reply:
Yes, that action also `uses node12 which is deprecated`.